lib/matplotlib/backend_bases.py

```diff
@@ -3363,9 +3586,14 @@ def remove_toolitem(self, name):
         raise NotImplementedError
 
 
+class ToolbarBase(ToolContainerBase, FlowBase):
```

Comment:
Nope, the ``ExpandableBase`` class only attaches to the ``FigureCanvasBase``, in your MEP22 branch you had Toolbar as non-expandable, I don't know how you set it before...